apps/storybook/package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "build": "build-storybook -c .storybook -o ./build",
     "dev": "start-storybook -p 6006 --quiet",
-    "clean": "rm -rf .turbo node_modules build"
+    "clean": "rm -rf .turbo node_modules build",
+    "percy": "percy storybook ./build",
```

Comment:
Runs against static Storybook build instead of local or live Storybook URL

Reply:
could we hoist this command to the root? would make the calling the script easier.